src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -0,0 +1,307 @@
+                    disabled={true}
+                />
+            );
+        } else if (selectedProperty.data.type === 'string') {
+            return (
+                <TextField
+                    styles={classNames.subComponentStyles.valueField}
+                    onChange={onChangeValueField}
+                />
+            );
+        } else if (selectedProperty.data.type === 'boolean') {
+            return (
+                <Dropdown
+                    options={[
+                        {
+                            key: 'True',
+                            text: 'True',
+                            selected: true
+                        },
+                        {
+                            key: 'False',
+                            text: 'False'
+                        }
+                    ]}
+                    onChange={onChangeDropdownValue}
+                    styles={classNames.subComponentStyles.valueField}
+                />
+            );
+        } else {
+            return (
+                <SpinButton
+                    styles={classNames.subComponentStyles.valueField}
+                    onChange={onChangeValueField}
+                />
+            );
+        }
+    };
+
+    return (
+        <div className={classNames.root}>
+            {!isRemoveDisabled && (
+                <div className={classNames.firstColumn}>
+                    {position !== 0 && (
+                        <Dropdown
+                            options={[
+                                {
+                                    key: 'And',
+                                    text: 'And',
+                                    selected: true
+                                },
+                                {
+                                    key: 'Or',
+                                    text: 'Or'
+                                }
+                            ]}
+                            onChange={onChangeCombinator}
+                        />
+                    )}
+                </div>
+            )}
+            <div className={classNames.inputColumn}>
+                {/* Property */}
+                <Select<PropertyOption>
+                    id={propertySelectorId}
+                    options={propertyOptions}
+                    noOptionsMessage={() =>
+                        t('advancedSearch.noPropertiesFound')
+                    }
+                    isSearchable={true}
+                    components={{ Group: Group, Menu: Menu }}
+                    onChange={onChangePropertySelected}
+                    styles={propertySelectorStyles}
+                />
+            </div>
+            <div className={classNames.inputColumn}>
+                {/* Operator */}
+                <Dropdown
+                    options={operatorOptions}
+                    selectedKey={selectedOperator ? selectedOperator.key : null}
```

Comment:
this should have placeholder values. `Select an operator` or `Select operator` and if a property is not selected it should say `No property selected`